crowsnest/parameter.py

```
@@ -0,0 +1,29 @@
+
+
+class Parameter:
+    def __init__(self, type=str, default=None) -> None:
+        self.type = type
+        self.set_value(default)
+
+    def set_value(self, value):
+        try:
+            if value is None:
+                self.value = None
+            elif self.type == bool:
```

Ruff (0.14.14)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)